js/choices.js

```diff
@@ -4238,7 +4248,7 @@
             if (!items.length || !this.config.removeItems || !this.config.removeItemButton) {
                 return;
             }
-            var id = element && parseDataSetId(element.parentElement);
+            var id = element && parseDataSetId(element.closest('[data-id]'));
```

Comment:
Where does this change come from? It is not in the original code.

<img width="645" height="289" alt="" src="https://github.com/user-attachments/assets/4f94e768-d41b-4729-a480-b84079887087" />